CMakeLists.txt

```diff
@@ -89,8 +89,21 @@ add_subdirectory(pennylane_lightning/core/src)
 #####################################################
 add_library(pennylane_lightning INTERFACE)
 
+target_link_libraries(pennylane_lightning INTERFACE lightning_observables
+                                                    lightning_utils
+                                                    lightning_algorithms
+                                                    )
+
+target_link_libraries(pennylane_lightning INTERFACE  ${PL_BACKEND} #simulator
+                                                    "${PL_BACKEND}_algorithms"
+                                                    "${PL_BACKEND}_observables"
+                                                    "${PL_BACKEND}_bindings"
+                                                    "${PL_BACKEND}_measurements"
+)
+                                                
 target_include_directories(pennylane_lightning  INTERFACE "$<INSTALL_INTERFACE:${PROJECT_SOURCE_DIR}/pennylane_lightning/src;include>")
```

Comment:
Thanks @maliasadi! Could you please also change `pennylane_lightning/src` to `pennylane_lightning/core/src` here?

Reply:
Thanks @multiphaseCFD! Fixed [a7fcf1d](https://github.com/PennyLaneAI/pennylane-lightning/pull/482/commits/a7fcf1dafe848417c8d535b2c838af709e732a98)